### DISCUSSION TITLE
Ability to share links

### DISCUSSION THREAD

**Discussion by Bonez0r** [schlagmichdoch/PairDrop#136] [Ideas]
I don't know how feasible this is, but in addition to sharing files I'm also looking for a feature that lets me share links. 

I used to have an app that copied clipboards between my smartphone and PC, but the app stopped working. It would be nice to have the feature here in Pairdrop. Or instead of sharing the clipboard, just sharing the link as a string of text is fine too. If this can't be done, then suggestions of good apps that do this are also welcome.

**Comment by schlagmichdoch** [comment]
This should already work as of now. What mobile OS do you use? Simply install PairDrop as a PWA, use the share menu to share links via PairDrop and select your recipient.

**Comment by ann0see** [comment]
In Snapdrop - and probably also PairDrop you can just long click on the device and you should be able to send a text message.

**Comment by schlagmichdoch** [comment]
For everyone who hosts PairDrop themselves, I might add that this feature probably only works when your instance is served via HTTPS to properly install it as PWA. See the [Deployment Notes](https://github.com/schlagmichdoch/PairDrop/blob/master/docs/host-your-own.md#deployment-notes) for more information!

**Comment by Bonez0r** [comment]
Omg I must be blind, it says right there in the app and web page how to do it. Sorry for this thread. Love the app.

**Comment by Today20092** [comment]
I know this is an old thread, but have you looked at KDE Connect for your phone?
https://alternativeto.net/software/kde-connect/about/
https://kdeconnect.kde.org/
